tools/dumppdf.py

```diff
@@ -128,51 +129,186 @@ def dumpallobjs(out, doc, mode=None):
     out.write('</pdf>')
     return
 
-# dumpoutline
+
+def resolve_dest(dest, doc):
```

Comment:
Would be nice to have some documentation describing what this does and what arguments it expects, and what it returns

Reply:
This is not a function we changed/refactored